hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/NameNodeExecutor.java

```diff
@@ -141,4 +147,18 @@ public void frameworkMessage(ExecutorDriver driver, byte[] msg) {
         .build());
     }
   }
+
```

Comment:
I think this check may be useful for a higher level of process supervision, although I'm not clear on why it is needed if our logic is otherwise correct and we have the proper shutdown hook. If we do add this check, we need to add it for all task types for consistency, not just ZKFC